drivers/circutor_cdp_duo.json

```diff
@@ -3,28 +3,28 @@
-    "grid_out_P_total": {"register": "0x72", "description": "Total active power measured in the load (sign indicates consumer or generated)", "unit": "W", "words": 2, "datatype": "int32", "multiplier": 1},
-    "pv_Pac_L1": {"register": "0x74", "description": "L1 active power measured in the pv (sign indicates consumer or generated)", "unit": "W", "words": 2, "datatype": "int32", "multiplier": 1},
-    "pv_Pac_L2": {"register": "0x76", "description": "L2 active power measured in the pv (sign indicates consumer or generated)", "unit": "W", "words": 2, "datatype": "int32", "multiplier": 1},
-    "pv_Pac_L3": {"register": "0x78", "description": "L3 active power measured in the pv (sign indicates consumer or generated)", "unit": "W", "words": 2, "datatype": "int32", "multiplier": 1},
-    "pv_Pac_total": {"register": "0x7A", "description": "Total active power measured in the pv (sign indicates consumer or generated)", "unit": "W", "words": 2, "datatype": "int32", "multiplier": 1},
-    "reg_perc_L1": {"register": "0x7D", "description": "L1 regulation percentage", "unit": "%", "words": 1, "datatype": "uint16", "multiplier": 1},
-    "reg_perc_L2": {"register": "0x7E", "description": "L2 regulation percentage", "unit": "%", "words": 1, "datatype": "uint16", "multiplier": 1},
-    "reg_perc_L3": {"register": "0x7F", "description": "L3 regulation percentage", "unit": "%", "words": 1, "datatype": "uint16", "multiplier": 1},
-    "reg_perc_total": {"register": "0x7C", "description": "Global regulation percentage", "unit": "%", "words": 1, "datatype": "uint16", "multiplier": 1},
-    "grid_out_E": {"register": "0x80", "description": "Active three-phase energy measured in the load", "unit": "Wh", "words": 2, "datatype": "uint32", "multiplier": 1},
-    "grid_in_E": {"register": "0x82", "description": "Active three-phase energy consumed from the grid", "unit": "Wh", "words": 2, "datatype": "uint32", "multiplier": 1},
-    "pv_E_dc": {"register": "0x84", "description": "Three-phase active photovoltaic energy", "unit": "Wh", "words": 2, "datatype": "uint32", "multiplier": 1},
-    "feed_in_E": {"register": "0x86", "description": "Active three-phase energy injected into the grid", "unit": "Wh", "words": 2, "datatype": "uint32", "multiplier": 1},
-    "grid_out_freq": {"register": "0x96", "description": "Frequency load", "unit": "Hz", "words": 2, "datatype": "uint32", "multiplier": 0.1},
-    "grid_in_freq": {"register": "0x14C", "description": "Frequency grid", "unit": "Hz", "words": 2, "datatype": "uint32", "multiplier": 0.1},
-    "pv_freq": {"register": "0x1B0", "description": "Frequency pv", "unit": "Hz", "words": 2, "datatype": "uint32", "multiplier": 0.1}
+    "grid_in_P_L1": {"register": "100", "description": "L1 Active power of the Grid (sign indicates consumed or generated)", "unit": "W", "words": 2, "datatype": "int32", "multiplier": 1},
```

Comment:
If you do want to convert to decimal registers, then they should not be quoted - i.e. numbers and not strings. (but per my other comment, I'm not sure that's the issue here)